specification/2.0/README.md

```diff
@@ -837,19 +837,44 @@ After applying morph targets to vertex positions and normals, tangent space may
 
 ### Skins
 
-All skins are stored in the `skins` array of the asset. Each skin is defined by the `inverseBindMatrices` property (which points to an accessor with IBM data), used to bring coordinates being skinned into the same space as each joint; and a `joints` array property that lists the nodes indices used as joints to animate the skin. The order of joints is defined in the `skin.joints` array and it must match the order of `inverseBindMatrices` data. The `skeleton` property points to node that is the root of a joints hierarchy. 
+All skins are stored in the `skins` array of the asset. Each skin is defined by the `inverseBindMatrices` property (which points to an accessor with IBM data), used to bring coordinates being skinned into the same space as each joint; and a `joints` array property that lists the nodes indices used as joints to animate the skin. The order of joints is defined in the `skin.joints` array and it must match the order of `inverseBindMatrices` data. The `skeleton` property points to the node that is the root of a joints hierarchy. 
 
-> **Implementation Note:** Matrix, defining how to pose the skin's geometry for use with the joints ("Bind Shape Matrix") should be premultiplied to mesh data or to Inverse Bind Matrices. 
+> **Implementation Note:** The matrix defining how to pose the skin's geometry for use with the joints ("Bind Shape Matrix") should be premultiplied to mesh data or to Inverse Bind Matrices. 
+
+> **Implementation Note:** Client implementations should apply only the transform of the skeleton root node to the skinned mesh while ignoring the transform of the skinned mesh node. In the example below, the position of `node_0` and the scale of `node_1` are applied while the position of `node_3` and rotation of `node_4` are ignored.
```

Comment:
/cc @takahirox this is consistent with the three.js defaults, I think?

Reply:
I think yes if `THREE.SkinnedMesh.bindMode` is `attached`.

So, this means skeleton can't be shared among multiple skinned mesh nodes in glTF as with Three.js?

Reply:
The skeleton can be shared by skinned mesh nodes that all inherit the transform of the skeleton root. The [Brainstem](https://gltf-viewer.donmccurdy.com/#model=https://raw.githubusercontent.com/KhronosGroup/glTF-Sample-Models/master/2.0/BrainStem/glTF/BrainStem.gltf) model is an example, with 59 SkinnedMeshes all composing a single figure (open JS console in the link to see structure).

But having multiple distinct figures share the same skeleton, like https://github.com/mrdoob/three.js/pull/11666, is not supported in glTF with this clarification, as far as I can tell.

Reply:
> having multiple distinct figures share the same skeleton

Yeah, what I pointed was this case